snips_nlu/dataset/intent.py

```diff
@@ -0,0 +1,317 @@
+from __future__ import absolute_import, print_function
+
+from abc import ABCMeta, abstractmethod
+from builtins import object
+from pathlib import Path
+
+from deprecation import deprecated
+from future.utils import with_metaclass
+
+from snips_nlu.__about__ import __version__
+from snips_nlu.constants import DATA, ENTITY, SLOT_NAME, TEXT, UTTERANCES
+
+
+class IntentFormatError(TypeError):
+    pass
+
+
+INTENT_FORMATTING_ERROR = IntentFormatError(
```

Comment:
I think it would be better to have a global variable for the exception message not for the exception object itself